index.html

```diff
@@ -1,220 +1,285 @@
-</div>
-<!-- About me -->
-<section class="MainAboutme">
-<div class="Aboutme">
-    <h3 class="mainText CreteRoundfont">About me</h3>
-<p class="textcolor Poppinsfont">
-    I’am a Software Developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-don’t hestiate to contact me.
-</p>
-<button type="button" class="PrjBTN SePrjBTN Interfont">
-    Get My Resume
-</button>
-</div>
-<hr class="aboutmeHr">
+      </ul>
+      <button type="button" class="PrjBTN Interfont">See Project</button>
+    </div>
+    <!-- About me -->
+    <section class="MainAboutme">
+      <div class="Aboutme">
+        <h3 class="mainText CreteRoundfont">About me</h3>
+        <p class="textcolor Poppinsfont">
+          I’am a Software Developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <button type="button" class="PrjBTN SePrjBTN Interfont">
+          Get My Resume
+        </button>
+      </div>
+      <hr class="aboutmeHr" />
 
-<ul class="nostyle boldfont">
-    <li> 
-        <img src="./images/Dimond.svg" alt="Diamond sign">Language
-        <ul class="nostyle"><li>Javascript</li>
+      <ul class="nostyle boldfont">
+        <li>
+          <img src="./images/Dimond.svg" alt="Diamond sign" />Language
+          <ul class="nostyle">
+            <li>Javascript</li>
             <li>Ruby</li>
             <li>Html</li>
             <li>Css</li>
-        </ul>
-    </li>
-    <li><img src="./images/Square.svg" alt="square sign">Frameworks
-        <ul class="nostyle">
+          </ul>
+        </li>
+        <li>
+          <img src="./images/Square.svg" alt="square sign" />Frameworks
+          <ul class="nostyle">
             <li>Bootstrap</li>
             <li>Ruby on Rails</li>
             <li>RSPec</li>
             <li>CapyBara</li>
             <li>Selenium</li>
-        </ul>
-    </li>
-    <li> <img src="./images/Circle.svg" alt="circle sign"> Skills
-        <ul class="nostyle">
+          </ul>
+        </li>
+        <li>
+          <img src="./images/Circle.svg" alt="circle sign" /> Skills
+          <ul class="nostyle">
             <li>Codekit</li>
             <li>Github</li>
             <li>Codepen</li>
             <li>Gitlab</li>
             <li>Terminal</li>
-        </ul>
-    </li>
-</ul>
-</section>
+          </ul>
+        </li>
+      </ul>
+    </section>
+
+    <footer class="footer">
+      <div class="Interfont footerTxt">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </div>
+      <form
+        class="contactForm"
+        action="https://formspree.io/f/xeqnpapb"
+        method="POST"
+      >
+        <input
+          class="no-outline"
```

Comment:
Kindly add some bottom padding to make it look clean and near to the required Figma design.